packages/big-design-icons/scripts/build-flags.js

```diff
@@ -55,7 +55,9 @@ async function generateFlags() {
     console.log(`Building: ${name}`);
 
     return convertToReactComponent(iconFilePath, name);
-  });
+  })) {
+    await buildPromise;
```

Comment:
Not sure of a better approach for looping through iterators. @deini do you have a better idea?

Reply:
Since we aren't using the result, what about something like:

```ts
async function generateIcons() {
  const iconFiles = await glob(SOURCE);

  // eslint-disable-next-line @typescript-eslint/no-unused-vars
  for await (const _result of asyncPool(cpus().length, iconFiles, (iconFilePath) => {
    const filename = basename(iconFilePath, '.svg');
    const name = `${camelcase(filename, { pascalCase: true })}Icon`;

    componentNames.add(name);

    // eslint-disable-next-line no-console
    console.log(`Building: ${name}`);

    return convertToReactComponent(iconFilePath, name);
  }));
}
```

Maybe moving the `asyncPool` call to a fn may make it easier to read too.

Reply:
I was trying to avoid an `eslint-disable` 😂  but good idea. Updated.